### DISCUSSION TITLE
Long Overdue Update

### DISCUSSION THREAD

**Discussion by Jyers** [Jyers/Marlin#1841] [General Discussion]
Hey all!

Sorry it's been so long once again... It seems almost cruel the way I left the "I'm back" discussion pinned throughout my latest absence. I'm not going to write a whole article as I did last time since the story is much the same. With school and other commitments I have not had much time to think on the firmware. To be honest, I haven't even had the time to print in the last 2 months. This combined with the dread of coming back to the firmware after leaving it for so long has kept me away. But this time I'm going to try to figure out a sustainable solution.

First off I'd like to thank all of you for your commitment to the project and for sticking with it throughout these gaps. I'd particularly like to thank @tititopher68-dev for his help throughout the repo and for attempting to keep the firmware updated.

Going forward I'm going to try to allow for more community development rather than simply issues and occasional pull requests. @tititopher68-dev has informed me that there is already an updated version with many new features and some new restructuring of the code. While I haven't seen any of it yet, I imagine this will be a good starting point for future development of the project.

You may see new branches, PRs, and updates in the coming days and weeks to setup the repo for better development. I hope you understand and support me in my attempts to keep the project alive as best I can.

As always, thank you all for your support and time :)

**Comment by K2Kevin** [comment]
Im not the only one that's going to say, there is **no need to apologize**.  LIFE always comes first!  Great to hear you are doing well, and all respect to you and the other developers that continue to move this firmware forward.

**Comment by VSMS6MRDR** [comment]
Yep, I do believe Kevin hit it on the mark! No other way to say it except how he did. Never need to apologize,those of us old enough to have had Life take over, feel your pain! Welcome back and all you do is appreciated.

**Comment by thedroidkid** [comment]


**Reply by mostlyflawles** [reply]
https://github.com/mriscoc/Ender3S1/wiki
I don't have any S1...

**Reply by ledcomp** [reply]
I would never use his software again. I said on reddit that the version of his software I used was buggy. Next thing I see a message from his reddit moderator (aka him) and I was banned from his forum. He also refuses to share his source code. This guy can not be trusted.

**Reply by zierbeek** [reply]
apparently his new firmware, professional firmware, is compliant, according to tinkyhead
https://github.com/mriscoc/Ender3V2S1/issues/2

**Reply by ctsvnino** [reply]
What's the link to join the discord?

**Reply by zierbeek** [reply]
@tititopher68-dev allright, now I see the whole story, thanks for the heads-up!

**Reply by ledcomp** [reply]
Now "mr professional" banned me from commenting on his github page, like I would want to. I noticed the comment box was greyed out. lol

**Reply by zierbeek** [reply]
same happened to me haha!

**Reply by ledcomp** [reply]
@zierbeek and @thedroidkid Welcome to the exclusive members only club. Pull up a chair and have a drink. Why would anyone trust hidden firmware from someone that bans people for asking questions?

**Reply by ledcomp** [reply]
Agreed... Thanks again

**Comment by Sbliss0930** [comment]
I'm sorry and I dont even know if I'm typing in the correct place however I very much so appreciate the firmware that I'm currently using on my ender 3 v2 with Bltouch and all metal microswiss direct drive hotend. The Jyers firmware has made my experience just so much better with my machine.  However I cant figure out how to get higher print temps with this firmware. I am not super tech savvy but hope you can show me where to go to get the info I need or new firmware I need to get higher print temps for doing carbon x filament.  Thanks for all the help in advance and thanks for the current firmware.

**Reply by AmonM** [reply]
https://github.com/MarlinFirmware/Marlin/issues/14993#issuecomment-522421855

Does this help you? You'll need to build your own firmware. If you'll need more help I could try helping you but I'm kinda new to building my own firmware too.
